blender_3dmigoto.py

```diff
@@ -2980,8 +2991,8 @@ def menu_func_import_pose(self, context):
     self.layout.operator(Import3DMigotoPose.bl_idname, text="3DMigoto pose (.txt)")
 
 def menu_func_export(self, context):
-    self.layout.operator(Export3DMigoto.bl_idname, text="3DMigoto raw buffers (.vb + .ib)")
-
+    self.layout.operator(Export3DMigoto.bl_idname, text="3DMigoto Mod")
+    
```

Comment:
Don't rename this please. Maybe if this is going to be more restrictive then the simple export case that might be a good case for adding it as a second option. I think you might be able to use inheritance to avoid duplicate code in the second operator if you do so.

Reply:
Also, there's whitespace on the blank line that should be removed.